components/02-molecules/alert/_yds-alert.scss

```diff
@@ -24,18 +26,31 @@ $alert-themes: map.deep-get(sass-tokens.$tokens, color, alert);
   }
 
   &[data-alert-state='dismissed'] {
-    display: none;
+    @include tokens.animate(opacity, $alert-fade-speed);
+
+    opacity: 0;
   }
 }
 
 .alert__inner {
   display: flex;
   align-items: flex-start;
   padding-block: var(--size-spacing-5);
+  height: 100%;
+  max-height: 7rem;
+  opacity: 1;
 
   @media (min-width: tokens.$break-mobile) {
     align-items: center;
   }
+
+  [data-alert-state='dismissed'] & {
+    @include tokens.animate(all, $alert-fade-speed);
+
+    opacity: 0;
+    max-height: 0;
+    padding: 0;
+  }
```

Comment:
This seems to work, but I'm curious why you added this all to the "inner" element, instead of to the main `.alert` element. I think it'd have added it all to the outer-most element.

Also, the `animate` mixin uses named arguments, so if you just want to affect the speed, you can do that like this
```
@include tokens.animate($duration: $alert-fade-speed);
```

Reply:
@ModulesUnraveled :thinking: It was working for me without using `$duration:`. I have added it.